fivetran_provider_async/operators.py

```diff
@@ -100,8 +101,21 @@ def execute(self, context: Context) -> None | str:
                     ),
                     method_name="execute_complete",
                 )
+            return None
 
     def _get_hook(self) -> FivetranHook:
+        import warnings
+
+        warnings.warn(
```

Comment:
_get_hook is private so should be ok to remove it

Reply:
Done.